### PR TITLE
Optimise for docker.

### PR DESCRIPTION
**Pull request by RealistikDash**
One small step for Performance Service, one giant leap for RealistikOsu.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
-REDIS_URL=
+
+MYSQL_USER=
+MYSQL_PASSWORD=
+MYSQL_HOST=localhost
+MYSQL_PORT=3306
+MYSQL_DATABASE=rosu
+
+AMPQ_USER=
+AMPQ_PASSWORD=
```
Comment (7ez) [inline .env.example:10]: AMQP

File: src/config.rs [hunk 2]
```diff
@@ -7,13 +7,46 @@ pub struct Config {
...
+    #[clap(long, env)]
+    pub mysql_port: u16,
+
+    #[clap(long, env)]
+    pub mysql_database: String,
+
+    #[clap(long, env)]
+    pub ampq_user: String,
+
+    #[clap(long, env)]
```
Comment (7ez) [inline src/config.rs:27]: amqp

**Review by 7ez — CHANGES_REQUESTED** [review]

**Comment by RealistikDash** [comment]
Oh didnt expect you here!

**Comment by RealistikDash** [comment]
Unlucky ig.

**Pull request merged**